DOCSP-26370: FAQ entry for MongoSecurityException (#427)
mongodb/docs-java: source/faq.txt

   :ref:`API documentation <java-api-landing>`.

How do I prevent the "com.mongodb.MongoSecurityException" error?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Your application may throw this exception if you attempt to connect to

Reviewer: I: [may/might](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#m:~:text=than%20matrices.-,may,-Use%20may%20to)

```suggestion
Your application might throw this exception if you attempt to connect to
```

> Your application may throw this exception if you attempt to connect to
> a MongoDB deployment by specifying invalid or incorrectly formatted credentials.

Reviewer: S: simplification

```suggestion
Your application may throw this exception if you specify invalid or incorrectly formatted credentials
when connecting to a MongoDB deployment.
```

If you receive this error when you attempt to connect to a MongoDB deployment,
check the following items in your code:

- The connection URI corresponds to the correct MongoDB deployment.
  To learn more about setting your connection URI, see :ref:`connection-uri`.

> - The credentials for the authentication mechanism that you specified are

Reviewer: S: On line 191, you leave out "that" in the first sentence. Suggest being consistent across list items.
